esp/scm/ws_config2.ecm

```diff
@@ -376,6 +409,10 @@ ESPservice [auth_feature("DEFERRED"),version("2.0"), default_client_version("2.0
         [
             description("Return the tree of nodes beneath the indicated node for the indicated number of levels")
         ] GetNodeTree(GetTreeRequest, GetTreeResponse);
+    ESPMethod
+        [
+            description("Find nodes matching the indicated path")
+        ] FindNodes(FindNodesRequest, FindNodesResponse);
```

Comment:
FetchNodes?

Reply:
See previous comment. Leaving as find to keep same paradigm throughout all of config manager.